src/ipset.c

```diff
@@ -347,24 +346,32 @@ static int ipset_parse_conf_file(void)
         RTE_LOG(WARNING, IPSET, "no memory for ipset buf\n");
         return -1;
     }
+
+    while(!feof(g_current_stream))
+    {
+        if((ch=getc(g_current_stream))=='\n')
```

Comment:
Do we need to remove the blank lines? We may use a  "readline" function(such as "readn" in src/common.c) to get this.